### PR TITLE
Correct raw `ons_deaths` table names in documentation

### PR DESCRIPTION
**Pull request by StevenMaude**

### CODE REVIEW

File: docs/includes/generated_docs/schemas/beta.raw.emis.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ This table contains all registered deaths.
 The `ehrql.tables.beta.ons_deaths` table contains the earliest registered death.
 
 !!! tip
-    To return one row per patient from `ehrql.tables.beta.raw.ons_deaths`,
+    To return one row per patient from `ehrql.tables.beta.raw.core.ons_deaths`,
```
Comment (StevenMaude) [inline docs/includes/generated_docs/schemas/beta.raw.emis.md:42]: comment: Again, this gets inherited from the `core` table.

**Comment by StevenMaude** [comment]
Discussed with @inglesp and we agreed that this could be merged now. Even though the EMIS entry isn't strictly correct, it's currently completely incorrect, and _this PR improves the rest of the entries_.

I've opened an issue to discuss how we best override documentation for tables that are reused from `core`: #1837.

File: ehrql/tables/raw/core.py [hunk 2]
```diff
@@ -37,10 +37,10 @@ class ons_deaths_raw(EventFrame):
 
     In the associated database table a small number of patients have multiple registered deaths.
     This table contains all registered deaths.
-    The `ehrql.tables.ons_deaths` table contains the earliest registered death.
+    The `ehrql.tables.raw.core.ons_deaths` table contains the earliest registered death.
```
Comment (iaindillingham) [inline ehrql/tables/raw/core.py:40]: I think this should be...

```suggestion
    The `ehrql.tables.core.ons_deaths` table contains the earliest registered death.
```

**Review by iaindillingham — APPROVED** [review]
See comment, though 🙂

File: ehrql/tables/raw/core.py (hunk 2 above)
Comment (StevenMaude) [reply ehrql/tables/raw/core.py:40]: Yes, you're right, thanks!

**Pull request merged**